src/components/schedule-page/table/schedule/schedule-parts/base-cell/cell-details-content.component.tsx

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import React, { useContext } from "react";
+import { TranslationHelper } from "../../../../../../helpers/translations.helper";
+import { shifts } from "../../../../../../common-models/shift-info.model";
+import { MdClose } from "react-icons/md";
+import { ScheduleLogicContext } from "../../use-schedule-state";
+import { Sections } from "../../../../../../logic/providers/schedule-provider.model";
+import { ShiftsInfoLogic } from "../../../../../../logic/schedule-logic/shifts-info.logic";
+
+export interface CellDetailsOptions {
+  index: number;
+  day: number;
+  month: number;
+  year: number;
+  rowIndex: number;
+  shift: string;
+  sectionKey: string;
+  close: () => void;
+}
+
+export function CellDetails(props: CellDetailsOptions): JSX.Element {
+  const { index, day, month, year, shift, close, sectionKey, rowIndex } = props;
+  let displayedYear = year;
+  let monthName = `${TranslationHelper.polishMonthsGenetivus[month]}`;
+  if (index < day) {
+    if (month > 0) {
+      monthName = `${TranslationHelper.polishMonthsGenetivus[month - 1]}`;
+    } else {
+      monthName = `${TranslationHelper.polishMonthsGenetivus[month + 11]}`;
+      displayedYear = year - 1;
+    }
+  }
+  const shiftcode = shift;
```

Comment:
to nie wnosi nic poza zmianą nazwy, a używając operatora spread na obiekcie da się zmienić nazwę: 
https://flaviocopes.com/how-to-rename-object-destructuring/

Reply:
zmieniłam nazwę przychodzącego parametru na `shiftcode`